SCAutolib/models/file.py

```diff
@@ -261,6 +322,7 @@ def save(self):
                 # to empty parser object
                 self._changes = ConfigParser()
                 self._changes.optionxform = str
+        os.chmod(self._conf_file, 0o600)
```

Comment:
I'm not sure if this should be done here or directly in controller. But if it happens here, docstring needs to be modified.

Reply:
It should be done here because after this method is called it is expected that the file is in a ready-to-use state. So, will add docstring